### ISSUE TITLE
Reload Checkout Is not being fired

### ISSUE THREAD

**Issue by christhebrews** [BlueAcornInc/UniversalAnalytics#14]
Hey,

Just a quick note, I'm on Magento 1.14.

You overrided the checkout.gotoSection function in the onestepcheckout.phtml file, but when you call the gotoSectionparent at the end of that you aren't including the reloadProgressBlock flag.  This is causing the checkout progress call to not be fired.

You should update line 72 in onestepcheckout.phtml from:

```javascript
this.gotoSectionparent(section);
```

 to:

```javascript
this.gotoSectionparent(section, reloadProgressBlock);
```

Thanks,
Chris

**Comment by Skrath** [comment]
Looks like that's something new that they've added, I'll have to get that in there. Thanks!

**Comment by Skrath** [comment]
This should be fixed via #15
